Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -358,7 +358,7 @@ class CallController {
     }
     
     private func handleReconnectionError() {
-        log.error("Error while reconnecting to the call")
+        log.warning("Error while reconnecting to the call")
```

Comment:
Is it possible to have the `log.error` with optional error? Since here `error` is more suitable than `warning`.